compiler/rustc_borrowck/src/nll.rs

```diff
@@ -273,7 +273,7 @@ pub(super) fn dump_nll_mir<'tcx>(
         regioncx.dump_graphviz_raw_constraints(&mut file)?;
     };
 
-    // Also dump the inference graph constraints as a graphviz file.
+    // Also dump the inference graph constraints SCCs as a graphviz file.
```

Comment:
why not just also call it "region constraint SCC graph" here?

Reply:
Sure.